app/models/views/summary.rb

```diff
@@ -34,6 +35,12 @@ def initialize(session)
       end
     end
 
+    def fee_paid
+      fee = I18n.t("fee_paid_#{@fee_paid}", scope: 'summary')
+      fee += ", on #{Date.parse(@fee_date_paid).strftime('%e %b %Y')}" if @fee_date_paid
+      fee
```

Comment:
I had to go with 
```ruby
def payment_date
  ", on #{Date.parse(@fee_date_paid)}" if @fee_date_paid
end
```
as Date.parse failed when @fee_date_paid was, legitimately nil.

The format was being set by the date/time config settings.